recipes/capnproto/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        with tools.run_environment(self):
```

Comment:
This is a bigger problem with conan and the cci recipes: once one of the requirements of a installer_package is built as a shared library, the executable won't run inside cmake's `add_custom_command`.